common/api/metadata.go

```diff
@@ -127,6 +127,8 @@ var (
 		"ListSchedules":                      {Scope: ScopeNamespace, Access: AccessReadOnly},
 		"UpdateWorkerBuildIdCompatibility":   {Scope: ScopeNamespace, Access: AccessWrite},
 		"GetWorkerBuildIdCompatibility":      {Scope: ScopeNamespace, Access: AccessReadOnly},
+		"UpdateWorkerVersioningRules":   	  {Scope: ScopeNamespace, Access: AccessWrite},
```

Comment:
can you just fix this indentation? (run gofmt/goimports on it or just open/save in your editor probably does it)

(and other files too)